qiita_pet/support_files/doc/source/processingdata/deblur_2021.09.rst

```diff
@@ -0,0 +1,42 @@
+deblur version 2021.09
+======================
+
+The deblur version 2021.09 addresses a bug with the fragment insertion parsing and
+cache that ignored some fragments for getting an accurate placement in the tree. In
+summary, in some occasions SEPP will return multiple fragments in a single entry; which
+was unexpected by the qp-deblur plugin parser, which assumed only one entry - the
+extra features will be seen as missing by the plugin and that information was
+sent and stored in the cache provided by Qiita, then propagated to future studies and
+meta-analyses.
+
+This bug was resolved in this `pull request <https://github.com/qiita-spots/qp-deblur/pull/60>`__.
+
+Sample counts implications
+--------------------------
+
+At the time of writing Qiita had 978,052 16S deblured private or pubic samples.
```

Comment:
unclear if you refer to the time of writing the bogus parser or this text

Reply:
this text, I can add more text to make it clearer ...